xep-0184.xml

```diff
@@ -203,7 +209,7 @@
   <received xmlns='urn:xmpp:receipts' id='richard2-4.1.247'/>
 </message>
 ]]></example>
-  <p>When the recipient sends an ack message, it SHOULD ensure that the message stanza contains only one child element, namely the &lt;received/&gt; element qualified by the 'urn:xmpp:receipts' namespace. In addition, it SHOULD include an 'id' attribute that echoes the 'id' attribute of the content message. Naturally, intermediate entities might add other extension elements to the message when routing or delivering the receipt message, e.g., a &lt;delay/&gt; element as specified in &xep0203;.</p>
+  <p>When the recipient sends an ack message, it SHOULD ensure that the message stanza contains only one child element, namely the &lt;received/&gt; element qualified by the 'urn:xmpp:receipts' namespace. In addition, it MUST include an 'id' attribute that echoes the 'id' attribute of the content message. Naturally, intermediate entities might add other extension elements to the message when routing or delivering the receipt message, e.g., a &lt;delay/&gt; element as specified in &xep0203;.</p>
```

Comment:
I suggest instead of "namespace. In addition, it MUST" we simply use ", which MUST". Otherwise it's unclear if the stanza or payload must have the id.

Reply:
Thanks for the suggestion.